Tests/LibWeb/Text/input/Editing/execCommand-selectAll.html

```diff
@@ -37,7 +37,6 @@
         // Report whether command is enabled and make sure it does not crash
         const documents = [
             document,
-            new Document(),
```

Comment:
You've removed the test that checked whether calling `selectAll` (which was a special case) crashed when called on an XMLDocument. Could you reintroduce that in `non-html-document.html`?

Reply:
I'd rather not change the behavior of an imported test if I can avoid it. I'll reintroduce the test I deleted here instead.

Reply:
Oh right! Missed that it was imported :-)